code/modules/admin/view_variables/filterrific.dm

```diff
@@ -0,0 +1,103 @@
+/datum/filter_editor
+	var/atom/target
+
+/datum/filter_editor/New(atom/target)
+	src.target = target
+
+/datum/filter_editor/ui_state(mob/user)
+	return GLOB.admin_state
+
+/datum/filter_editor/ui_interact(mob/user, datum/tgui/ui)
+	ui = SStgui.try_update_ui(user, src, ui)
+	if(!ui)
+		ui = new(user, src, "Filteriffic", "Filteriffic")
```

Comment:
Would be better if you defined this title on the window itself, via `<Window title="">` prop.